debian/enterprise-control-center/include/etc/confluent/docker/control-center.properties.template

```diff
@@ -7,6 +7,9 @@ confluent.monitoring.interceptor.topic.replication={{env.get('CONTROL_CENTER_MON
 confluent.controlcenter.internal.topics.replication={{env.get('CONTROL_CENTER_INTERNAL_TOPICS_REPLICATION', env['CONTROL_CENTER_REPLICATION_FACTOR'])}}
 confluent.controlcenter.command.topic.replication={{env.get('CONTROL_CENTER_COMMAND_TOPIC_REPLICATION', env['CONTROL_CENTER_REPLICATION_FACTOR'])}}
 
+confluent.metrics.topic.partitions={{env.get('CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS', env['CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS'])}}
```

Comment:
might be nice to let these have their own env var instead of piggybacking on `CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS` and `CONTROL_CENTER_REPLICATION_FACTOR`. something like `CONFLUENT_METRICS_TOPIC_{PARTITIONS,REPLICATION}` to match the config keys.